e2e-playwright/src/page-objects/product-detail.po.ts

```diff
@@ -6,7 +6,7 @@ export class ProductDetailPage {
     }
 
     async getTitle(): Promise<string> {
-        const title = await this.page.textContent('app-root .product-details h2');
+        const title = await this.page.textContent('app-product-detail .product-details h2');
         if (title === null) {
             throw 'Could not find detail page title tag';
```

Comment:
Das würde ich wie gesagt nicht machen.

Reply:
Mhh, ich dachte das Problem sei, dass wir nicht wissen, in welchem Component sich `app-product-detail` befindet. Daher überprüfe ich nicht mehr ob das in `app-root` oder sonstwo eingebunden ist.
Stattdessen garantiert `app-product-detail` doch, dass wir auf den Titel im korrekten Component zugreifen. Und da das die Kennung des aktuell zu testenden Components ist, haben wir da auch keine ungewollte Dependency hardcoded.

Reply:
Ich glaub das ist ein Missverständnis. Mein Kommentar bezoge sich auf Zeile 11 mit dem `throw`. Ich würde in den POs keine Errors werfen.

Reply:
Stimmt, das war ein Missverständnis. Ist erledigt.